package.json

```diff
@@ -87,19 +77,16 @@
     "snyk-protect": "snyk protect",
     "snyk": "snyk",
     "start": "node ./tools/dev-server.js",
-    "test:build": "mocha --timeout 60000 ./tests/test-build.js",
+    "test:build": "jasmine --random=false ./tests/test-build.js",
     "test:headless": "node ./tests/headless.js",
-    "test:karma": "npm run karma",
-    "postinstall": "opencollective-postinstall"
+    "test:karma": "npm run karma"
   },
   "dependencies": {
     "colors": "^1.4.0",
     "command-line-args": "^5.1.1",
     "command-line-usage": "^6.1.0",
     "epipebomb": "^1.0.0",
```

Comment:
We might not need this anymore in newer node:
https://github.com/mhart/epipebomb/issues/10

Reply:
We're supporting Node 12 presently though, as 14 isn't LTS yet.

Reply:
ah... true! Booooo!!!!